makefiles/kconfig.mk

```diff
@@ -88,11 +98,17 @@ USEPKG_W_PREFIX = $(addprefix PKG_,$(USEPKG))
 # defining symbols like 'MODULE_<MODULE_NAME>' or PKG_<PACKAGE_NAME> which
 # default to 'y'. Then, every module and package Kconfig menu will depend on
 # that symbol being set to show its options.
+# Do nothing when testing Kconfig module dependency modelling.
+ifeq (1,$(TEST_KCONFIG))
+$(KCONFIG_GENERATED_DEPENDENCIES): FORCE | $(GENERATED_DIR)
+	$(Q)touch $@
```

Comment:
Hmm, do we really need to touch the files? Wouldn't this also work without updating the modification date? Or will `make` just skip the full compilation phase?

Reply:
I did this because the file needs to be there as the root Kconfig file includes it. I'll see if it works just by making that inclusion optional.

Reply:
Hmm it was optional already, the problem was it was being wrongly generated.

Reply:
I removed the `touch`